src/app/api/auth/token/route.ts

```diff
@@ -47,4 +48,13 @@ export async function POST(request: NextRequest): Promise<Response> {
         }
     }
     return new Response(JSON.stringify({ response: ERROR_MESSAGES.unauthorized }), { status: 401 });
+}
+
+export async function GET(): Promise<Response> {
```

Comment:
These should have more descriptive function names. Also I don't think Next.js limits API routing by the function name and you'll have to actually check the request method.

https://nextjs.org/docs/pages/building-your-application/routing/api-routes

Reply:
I think that's only for the old convention? I'm using the route handler method.

https://nextjs.org/docs/app/building-your-application/routing/route-handlers